packages/webcrypto-ed25519-polyfill/src/__tests__/secrets-test.ts

```diff
@@ -44,6 +51,27 @@ describe('exportKeyPolyfill', () => {
     });
 });
 
+describe('importKeyPolyfill', () => {
+    const PKCS8_HEADER = new Uint8Array([
+        0x30, 0x2e, 0x02, 0x01, 0x00, 0x30, 0x05, 0x06, 0x03, 0x2b, 0x65, 0x70, 0x04, 0x22, 0x04, 0x20,
+    ]);
```

Comment:
Could you format and comment this the same as https://github.com/solana-labs/solana-web3.js/blob/eb5fd160f78c17f360d679861f487b42c693bd69/packages/keys/src/__tests__/signatures-test.ts#L11-L45 please? It'll make this test much easier to maintain. You can probably just copy-paste from there exactly if you're not already using the same header.